core/src/unprocessed_packet_batches.rs

```diff
@@ -39,6 +39,8 @@ pub enum DeserializedPacketError {
     SanitizeError(#[from] SanitizeError),
     #[error("transaction failed prioritization")]
     PrioritizationFailure,
+    #[error("vote transaction deserialization error")]
+    VoteTransactionError,
```

Comment:
Would vote transaction still land in unprocessed_packet_batches?

Reply:
no because the banking threads will never see this transaction